docker-compose-host.yml

```diff
@@ -71,6 +72,7 @@ services:
       - ${DEVSTACK_WORKSPACE}/src:/edx/src
       - ${PWD}/py_configuration_files/cms.py:/edx/app/edxapp/edx-platform/cms/envs/devstack.py
       - ${PWD}/py_configuration_files/lms.py:/edx/app/edxapp/edx-platform/lms/envs/devstack.py
+      - ${PWD}/py_private_requirements/cms.txt:/edx/private_requirements.txt
```

Comment:
[curious] Are there no `private_requirements.txt` for the LMS?

For example, is there an equivalent like the following needed:

```
${PWD}/py_private_requirements/lms.txt:/edx/private_requirements.txt
```

Reply:
Honestly, I don't know!  Like, why does the cms image need lms settings? Can't hurt to add both though.